app/models/miq_ae_domain.rb

```diff
@@ -100,6 +100,17 @@ def editable_property?(property)
 
   alias editable_contents? editable?
 
+  def update_attributes(hash)
```

Comment:
To follow Rails conventions the `update_attributes` method should only return `true` or `false` and an `update_attributes!` method would be used to raise errors.  And if possible can this be changed to use `update` and `update!`

```ruby
def update(hash)
  update!(hash)
rescue
  false
end

def update!(hash)
  not_editable = hash.keys.reject { |p| editable_property?(p) }
  ...
end

```

Reply:
@lfu @mkanoor Any thoughts on this change?  The change to use `update` depends on what the caller is using but we should be switching over to `update` anyway.